app/styles/templates/items/_detail.scss

```diff
@@ -15,6 +22,140 @@
 
   .search {
     margin-top: 0;
+
+    // --- HEADER IMAGE
+    #itemImage {
+      position: relative;
+      min-height: 4rem;
+
+      .slick-slider {
+        margin-bottom: 0;
+      }
+
+      // Black tranparent header
+      .image-overlay {
+        background-color: rgba(black, 0.6);
+        position: absolute;
+        bottom: 0;
+        width: 100%;
+        min-height: 4rem;
+        padding-top: 1rem;
+        padding-left: 1rem;
+        padding-right: 1rem;
```

Comment:
Can we combine this into `padding: 1rem 1rem 0` ?

Reply:
It's just a preference of mine, more descriptive. Plus, I can never remember the order in`padding: 1rem 1rem 0` 😆